config/lobby/db/001_create_tables

```diff
@@ -59,7 +59,7 @@ alter table muted_usernames owner to postgres;
 
 create table ta_users (
     username character varying(40) not null,
-    password character varying(40) not null,
+    password character varying(60) not null,
```

Comment:
Has @DanVanAtta already deployed Postgres in production?  If so, then this script can't be modified, and we'll need something like an `002_update_password_length` script to alter the table.